soc/intel/intel_adsp/tools/cavstool.py

```diff
@@ -249,14 +269,28 @@ def map_regs():
     global bar4_mmap
     (bar4_mem, bar4_mmap) = bar_map(pcidir, 4)
     dsp = Regs(bar4_mem)
-    dsp.ADSPCS         = 0x00004
-    dsp.HIPCTDR        = 0x00040 if cavs15 else 0x000c0
-    dsp.HIPCTDA        =                        0x000c4 # 1.8+ only
-    dsp.HIPCTDD        = 0x00044 if cavs15 else 0x000c8
-    dsp.HIPCIDR        = 0x00048 if cavs15 else 0x000d0
-    dsp.HIPCIDA        =                        0x000d4 # 1.8+ only
-    dsp.HIPCIDD        = 0x0004c if cavs15 else 0x000d8
-    dsp.SRAM_FW_STATUS = 0x80000 # Start of first SRAM window
+    if ace15 or ace20:
```

Comment:
Might as well use adsp_is_ace() for consistency, no?

Reply:
@andyross Fixed in updated push.